packages/seal/src/decrypt.ts

```diff
@@ -106,11 +99,6 @@ async function combine(shares: { index: number; share: Uint8Array }[]): Promise<
 
 	// The Shamir secret sharing library expects the index/x-coordinate to be at the end of the share
 	return externalCombine(
-		shares.map(({ index, share }) => {
-			const packedShare = new Uint8Array(share.length + 1);
-			packedShare.set(share, 0);
-			packedShare[share.length] = index;
-			return packedShare;
-		}),
+		shares.map(({ index, share }) => flatten([share, new Uint8Array([index])])),
```

Comment:
is `share` fixed size?

Reply:
Yes, it's a share of the base key which is fixed fo 32 bytes.